### PR TITLE
Shine Button Effects in Shine.less

### PR DESCRIPTION
**Pull request by hussein-hub**
Issue:

### CODE REVIEW

File: src/components/animated/_shine.less [hunk 1]
```diff
@@ -46,4 +113,17 @@
   &.orange-btn:active {
     .bc(@darkOrange);
   }
+  s &:active {
```
Comment (shahednasser) [inline src/components/animated/_shine.less:116]: why is there an s here?

File: src/components/animated/_shine.less [hunk 2]
```diff
@@ -46,4 +113,17 @@
...
+    &:after {
+      opacity: 0;
+    }
+  }
+}
+.shine-btn {
+  &:hover {
```
Comment (shahednasser) [inline src/components/animated/_shine.less:123]: nest in `.shine-btn` above

**Review by shahednasser — CHANGES_REQUESTED** [review]
It looks much better! I just have two comments below. Also please remove all the commented out code.

**Comment by hussein-hub** [comment]
Will do the mentioned changes by today

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @hussein-hub for code

**Pull request merged**